### DISCUSSION TITLE
Triggers based on day of the week ?

### DISCUSSION THREAD

**Discussion by SuperLanceur** [firefly-iii/firefly-iii#7982] [Firefly III] [answered]
Hello,
I'd like to trigger different rules based on the day of the week. 
For example, set the description to x if it was made on Monday and to y if it was made on Friday.
By looking at the documentation, I found this that is vaguely relevant, but it does not permit the actual behaviour I described.
https://docs.firefly-iii.org/firefly-iii/features/rules/#semi-specific-dates
Is it possible to do that somehow ?
Thanks in advance !

**Comment by JC5** [comment, marked as answer]
This is currently not possible, but I'll think about it!
